honeybee_radiance/modifier/material/mirror.py

```diff
@@ -0,0 +1,332 @@
+
+    Args:
+        identifier: Text string for a unique Material ID. Must not contain spaces
+            or special characters. This will be used to identify the object across
+            a model and in the exported Radiance files.
+        r_reflectance: Reflectance for red. The value should be between 0 and 1
+            (Default: 1).
+        g_reflectance: Reflectance for green. The value should be between 0 and 1
+            (Default: 1).
+        b_reflectance: Reflectance for blue. The value should be between 0 and 1
+            (Default: 1).
+        modifier: Material modifier (Default: "void").
+        alternate_material: An optional material may be used like the illum type to
+            specify a different material to be used for shading non-source rays. If this
+            alternate material is given as "void", then the mirror surface will be
+            invisible. This is only appropriate if the surface hides other (more
+            detailed) geometry with the same overall reflectance (Default: None).
+        dependencies: A list of primitives that this primitive depends on. This
+            argument is only useful for defining advanced primitives where the
+            primitive is defined based on other primitives (Default: []).
+
+    Properties:
+        * identifier
+        * display_name
+        * r_reflectance
+        * g_reflectance
+        * b_reflectance
+        * average_reflectance
+        * values
+        * modifier
+        * alternate_material
+        * dependencies
+        * is_modifier
+        * is_material
+
+    Usage:
+
+    .. code-block:: python
+
+        mirror_material = Mirror("mirror_material", 0.95, .95, .95)
+        print(mirror_material)
+    """
+
+    __slots__ = ('_r_reflectance', '_g_reflectance', '_b_reflectance',
+                 '_alternate_material',)
+
+    def __init__(
+        self, identifier, r_reflectance=1.0, g_reflectance=1.0, b_reflectance=1.0,
+            modifier='void', alternate_material=None, dependencies=None):
+        """Create mirror material."""
+        # add alternate material as a dependency if provided
+        self._alternate_material = None  # placeholder for alternate material
+        Material.__init__(self, identifier, modifier=modifier, dependencies=dependencies)
+        self.r_reflectance = r_reflectance
+        self.g_reflectance = g_reflectance
+        self.b_reflectance = b_reflectance
+        self.alternate_material = alternate_material
+        self._update_values()
+
+    def _update_values(self):
+        "update value dictionaries."
+        if self.alternate_material is not None:
+            self._values[0] = [self.alternate_material.identifier]
+        self._values[2] = [
+            self.r_reflectance, self.g_reflectance, self.b_reflectance,
+        ]
+
+    @property
+    def r_reflectance(self):
+        """Reflectance for red channel.
```

Comment:
It would be helpful to write these doctrings as:
```
Get or set reflectance for red channel.
```
This way, users will be able to tell from the online docs that this property is set-able.

Reply:
Thanks. Fixed.